impedance/models/circuits/elements.py

```diff
@@ -1,27 +1,27 @@
 import numpy as np
 
-initial_state = globals().copy()
-non_element_functions = ['element_metadata',
-                         'initial_state',
-                         'non_element_functions',
-                         'typeChecker',
-                         'circuit_elements']
-# populated at the end of the file -
-# this maps ex. 'R' to the function R to always give us a list of
-# active elements in any context
-circuit_elements = {}
+
+class ElementException(Exception):
```

Comment:
Thanks for adding these! I think it might be better to name them ElementError and OverwriteError (instead of ...Exception) to be more in line with the typical naming format of exceptions ending in Error

```suggestion
class ElementError(Exception):
```